CONTRIBUTE.md

```diff
@@ -52,7 +52,28 @@ reject comments, commits, code, wiki edits, issues, and other contributions
 that are not aligned to this Code of Conduct, or to ban temporarily or
 permanently any contributor for other behaviors that they deem inappropriate,
 threatening, offensive, or harmful.
-   
+
+## Guides for opening issues
+
```

Comment:
This should contain delated explanation.
Please refer to [this link](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md#issues) for more details

Reply:
Okay i have forked the repo again and this PR was just for one point out of the three. Will add the details.